gnomad_mitochondria/pipeline/add_annotations.py

```diff
@@ -142,13 +144,49 @@ def add_gnomad_metadata(input_mt: hl.MatrixTable) -> hl.MatrixTable:
     return input_mt
 
 
-def filter_by_copy_number(input_mt: hl.MatrixTable) -> hl.MatrixTable:
+def add_age_and_pop(input_mt: hl.MatrixTable, all_output: str) -> hl.MatrixTable:
+    """
+    Add sample-level metadata for age and pop.
+
+    If 'subset-to-gnomad-release' is set, these annotations are already added by the add_gnomad_metadata function.
+    If 'subset-to-gnomad-release' is not set, the user should include an 'age' and 'pop' column in the file supplied to `all-output`.
+
+    :param input_mt: MatrixTable
+    :param all_out: Path to metadata file downloaded from Terra
+    :return: MatrixTable with select age and pop annotations added
+    """
+    ht = hl.import_table(
+        all_output,
+        types={
+            "age": hl.tint32,
+            "pop": hl.tstr,
+        },
+        missing="NA",
+    ).key_by("s")
+
+    ht = ht.select(
+        "age",
+        "pop",
+    )
+
+    input_mt = input_mt.annotate_cols(**ht[input_mt.s])
```

Comment:
why use col_key instead?

Reply:
Either will work as expected if `s` is the key, but we typically use col_key, row_key, or key instead of the column name, it would allow the column name to be `subject_id` (or something else), but still work as expected as long as `ht` is keyed the way you want it to be